src/assets/sass/pages/_data.scss

```diff
@@ -14,3 +14,52 @@ adb-histogram .legend {
     margin: 5px;
   }
 }
+
+.spinner {
```

Comment:
Spinner css probably more correctly belongs in a `sass/components/_spinners.scss` file.